examples/aggregator/gcp/pod.yaml

```diff
@@ -24,6 +27,30 @@ spec:
       env:
         - name: RUST_LOG
           value: info
+    # Additional container to scrape prometheus metrics:
+    # https://github.com/GoogleCloudPlatform/k8s-stackdriver/tree/master/prometheus-to-sd
+    - name: prometheus-to-stackdriver
+      image: gcr.io/google-containers/prometheus-to-sd:v0.9.0
```

Comment:
FYI it looks like the latest version is 0.9.2, is there any reason why it was not chosen?

See https://pantheon.corp.google.com/gcr/images/google-containers/GLOBAL/prometheus-to-sd?gcrImageListsize=30